Source/WebCore/css/CSSRayValue.cpp

```diff
@@ -43,7 +46,9 @@ String CSSRayValue::customCSSText() const
         m_isContaining ? " contain"_s : ""_s,
         hasPosition ? " at "_s : ""_s,
         hasPosition ? m_position->cssText() : ""_s,
-        ')'
+        ')',
+        hasCustomCoordinateBox ? " "_s : ""_s,
+        hasCustomCoordinateBox ? nameString(toCSSValueID(m_coordinateBox)) : ""_s
```

Comment:
```suggestion
        hasCustomCoordinateBox ? nameLiteral(toCSSValueID(m_coordinateBox)) : ""_s
```

I think this is better (feel free to change before landing) since it avoids constructing a string.

Reply:
Fixed.